Pod/Classes/SwiftPhotoGallery.swift

```diff
@@ -245,8 +246,8 @@ public class SwiftPhotoGallery: UIViewController, UICollectionViewDataSource, UI
         let inspiratoBlue: UIColor = UIColor(red: 0.0, green: 0.66, blue: 0.875, alpha: 1.0)
         pageControl.currentPageIndicatorTintColor = inspiratoBlue
 
-        //let inspiratoBlueDim: UIColor = UIColor(red: 0.0, green: 0.66, blue: 0.875, alpha: 0.25)
-        //pageControl.pageIndicatorTintColor = inspiratoBlueDim
+        let dimGray: UIColor = UIColor(red: 0.75, green: 0.75, blue: 0.75, alpha: 0.35)
```

Comment:
Good to know! I'm going to make those paging control colors public facing in an upcoming rev so I'm not too concerned about it right now.